example/src/eyedrop/index.js

```diff
@@ -30,173 +31,182 @@ type Props = {
   onInit?: Function,
   onPickStart?: Function,
   onPickEnd?: Function,
-  passThrough?: string,
-  pickRadius?: { unit: 'pixel' | 'radius', amount: number }
+  colorsPassThrough?: string,
```

Comment:
More natural.
```suggestion
  passThroughColors?: string,
```

Reply:
The name is unclear both way. The name seems more like it's looking for a boolean than asking for a custom name for the color object. But I couldn't think of a better one so I just put "color" in there.

Reply:
Hmm... Yes, you're right.
It's a hard one to name.